core/src/Ledger.sol

```diff
@@ -112,15 +127,20 @@ contract Ledger {
         }
     }
 
+    /// @notice The number of shares held by `account`
+    function balanceOf(address account) external view returns (uint256) {
+        return balances[account] % Q112;
```

Comment:
mod basically just chops off higher order bits

Reply:
Why would we want to do this?

Reply:
Correct. The maximum `totalSupply` of banknotes (receipt tokens) is `2**112 - 1`, so each individual user's balance is guaranteed to fit in 112 bits.